WcaOnRails/app/models/inbox_person.rb

```diff
@@ -19,4 +19,15 @@ class InboxPerson < ApplicationRecord
   def country
     Country.find_by_iso2(countryId)
   end
+
+  # NOTE: silly method overriding: we don't have an id on that table.
+  # Hopefully this necessary dirty hack will go away when we streamline posting
+  # results through WCIF.
+  def delete
+    InboxPerson.where(id: id, competitionId: competitionId).delete_all
+  end
+
+  def update(args)
+    InboxPerson.where(id: id, competitionId: competitionId).update_all(args)
```

Comment:
Actually, despite using `update_all`, this is guaranteed to only update one record.
`InboxPerson` has a unique constraint on id+competitionId, but no actual id field (because id is a string and not an integer...), so this is the only way to update one record :(
(I know this is dirty and bad, but as said in the comment above this is gonna change soon)

Reply:
Sure, but it will still bypass validations due to using `update_all`, right?